src/pages/armDetail/armDetailView.js

```diff
@@ -58,14 +58,14 @@ const options = (classes) => ({
   filter: false,
   searchable: false,
   print: false,
-  download: true,
+  download: table.download,
   downloadOptions: {
-    filename: 'Bento_arm_files_download'.concat(dateTimeStamp()).concat('.csv'),
+    filename: table.downloadFileName.concat(dateTimeStamp()).concat('.csv'),
```

Comment:
Can you provide a default name if nothing is specified, otherwise it will through a null exception

Reply:
Good Point. I will update